pytest_reportportal/service.py

```diff
@@ -487,17 +463,15 @@ def get_marker_value(item, keyword):
                 if marker and marker.args else keyword
 
         try:
-            tags = [get_marker_value(item, k) for k in item.keywords
-                    if item.get_closest_marker(k) is not None
-                    and k not in self.ignored_tags]
+            get_marker = getattr(item, "get_closest_marker")
         except AttributeError:
-            # pytest < 3.6
-            tags = [get_marker_value(item, k) for k in item.keywords
-                    if item.get_marker(k) is not None
-                    and k not in self.ignored_tags]
-
-        tags.extend(item.session.config.getini('rp_tests_tags'))
+            get_marker = getattr(item, "get_marker")
+        tags = [{"key": "tag", "value": get_marker_value(item, k)}
```

Comment:
Actually you don't need to put key "tag" here. Tags - is an attribute without key.

Reply:
Honestly, it's not obvious. I'd recommend updating [DevGuide](https://github.com/reportportal/documentation/blob/master/src/md/src/DevGuides/reporting.md).